docs/topics/commands.rst

```diff
@@ -430,6 +430,8 @@ Supported options:
 * ``--callback`` or ``-c``: spider method to use as callback for parsing the
   response
 
+* ``--meta`` or ``-m``: additional request meta that will be pass to the callback request. This must be a valid json string. Example: --meta='{\"foo\" : \"bar\"}'
```

Comment:
could you please wrap this line?

Why is escaping of " needed?

Reply:
What exactly do you mean by wrap this line?
I was originally doing it like this --meta="'{\"foo\" : \"bar\"}" (escaping the "), but forgot you could just do it with ' so forgot to remove them. Sorry.

Reply:
We're wrapping lines in docs to some reasonable column limit (~80, not strict though), so that .rst files are readable without line-wrapping editors/viewers.